### DISCUSSION TITLE
Thank you Xodus Team

### DISCUSSION THREAD

**Discussion by bennysway** [JetBrains/xodus#106] [Show and tell]
Kudos to you guys for creating something really new and outstanding from the rest. Here's something I have made with the database (Compose Desktop)

![Dashboard Preview](https://tinypng.com/backend/opt/output/sf2q27gg73a51rfy4dchzc8w242a4ssg/miDash.jpg_large)
drive image link https://drive.google.com/file/d/17Et2RWadlHpQ84t-7zmlJH9ye2iZyBGz/view?usp=sharing

Its a medical platform and we are still in the product ideation and prototyping. Im an intermediate Kotlin developer and I would love to be a contributor so that my use case is covered. I saw the Youtrack tickets and the are solid. It would be good to replace the cassandra db underneath the api part of this project. Hence High Availability and Cloud deployment tasks are tickets Im watching. I am really a novice at times but with the right knowledge, isnt anything archivable? xD

In conclusion, thatnk you team. and I hope you reach extraodinary heights

**Comment by andrii0lomakin** [comment]
Hi @bennysway.
Thank you for using Xodus! 
We are glad to read it and do appreciate your feedback.

**Reply by bennysway** [reply]
Seeing the sunset was a bit sad but hey, if its evolution, I'm on board. Learning the green gremlin was a bit fun, but migration has started on my part. I have created custom mappers for edges and vertices, its now day 3 I'm discovering there is a mapper, the `AbstractIoRegistry` I have different project(s) now but will help or ask in zulip
